recipes-support/onboard/files/onboard_hover_seg_fault.patch

```diff
@@ -0,0 +1,37 @@
+--- a/Onboard/osk/osk_devices.c
++++ b/Onboard/osk/osk_devices.c
```

Comment:
This .patch is missing the commit message and .patch upstream status information.

Has this patch been submitted upstream?

Reply:
No, it hasn't been submitted upstream, I found this patch [here](https://github.com/void-linux/void-packages/commit/1be95325d320122efd5dedf7437839cfcca01f7a). The issue was reported in one of the forums, and this was the fix given, although the person hasn't submitted the fix upstream.

Reply:
OK. Then you can reference [this styleguide](https://github.com/ni/meta-nilrt/blob/nilrt/master/kirkstone/docs/styleguide.md#patch-files) on how to add the necessary summary and tracking information to the .patch. It's important that .patch files have good tracking information in them, so that rebasing maintainers know when to drop them from the recipe during upgrades.

Reply:
I have added the commit message from the source I got the patch file, and have added upstream status information as well.